Quote-API added

Some1Uknow opened the pull request:
# [New API] : Quote API #138

### Closes #138 

# 🗒️ Checklist:

- [] I have mentioned the issue number in my Pull Request.
- []  I have gone through `rules of when PR will get merged.` 
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`
- [] I have followed proper file structure.
- [] I have gone through [CONTRIBUTING GUIDELINES.md](../blob/master/CONTRIBUTING.md).

 Thank you for contributing!❤️ 

Don't forget to show some love by ⭐ the repository!

Some1Uknow committed:
add new api

Some1Uknow committed:
add new api

dishamodi0910 commented:
@Some1Uknow Please add more functionality so it can be labelled under level 3.

Some1Uknow committed:
New Features added

Some1Uknow commented:
Hey Disha, I have added many new features in the API (users can create new quotes, submit their own quote, rate a quote) and I have now added more than 50 quotes. In case any more changes are needed, then let me know.

dishamodi0910 commented:
@Some1Uknow We're thrilled to inform you that your pull request has been successfully merged into the APIverse🚀🎁
Your hard work, dedication, and expertise have made a significant impact on our project, and we couldn't be more grateful for your contributions.😇
Your code will now be a part of our growing ecosystem, helping developers worldwide to create exceptional APIs.❣️🔥
Happy Coding! ✨
Will See you soon❣️
